server/index.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const Sites = require('../db/Site.js');
+const Site = require('../db/Site.js');
```

Comment:
careful with changing Mongoose model names after creation, might break database access for others using your service

Reply:
model name is the same-- just that I had accidentally given it a different variable name when requiring it in my server.

Reply:
Ah, got it, I misread that.